src/clients/base-client.js

```diff
@@ -188,7 +188,7 @@ class BaseClient {
 
     if (this.authorizationRequired) {
       return this.tokenClient.createServiceToken({}).then(token => {
-        requestOptions.headers['authorization'] = `Bearer ${token.jwt}`
+        requestOptions.headers['Authorization'] = `Bearer ${token.jwt}`
```

Comment:
Why this change? Headers should be case-insensitive.
The error would be in the application that only accepts `Authorization` and not here.